time_tracker_jsnjack@gmail.com/extension.js

```diff
@@ -26,7 +26,11 @@ const TTNotificationBanner = new Lang.Class({
     addAction: function(label, callback) {
         // Style buttons
         var extra_style = "";
-        if (label === restart_button_name) {
+        
+        if (label === preferences_button_name) {
+            label = _("Preferences");
```

Comment:
It is because it is impossible to change label text : 
preferences_button_name = _("Preferences"),
restart_button_name = _("Restart"),
toggle_button_name = _("Pause/Resume");

directly with po/mo file, I had try, but it has no effect, i don't know why (maybe it is because it is set on the top of extension.js ?).

So I applied translation when labels are re-set (lines 31, 33, 37, 40) and it works.
So I added line 30, because "Preferences" was no re-set, and I could not translate it